packages/block-library/src/more/edit.native.js

```diff
@@ -47,29 +47,45 @@ export default class MoreEdit extends Component {
 		this.props.setAttributes( { customText: value } );
 	}
 
-	render() {
+	renderLine( key ) {
+		return <View key={ key } style={ styles[ 'block-library-more__line' ] } />
+	}
+
+	renderText( key ) {
 		const { attributes, onFocus, onBlur } = this.props;
 		const { customText } = attributes;
 		const defaultText = __( 'Read more' );
 		const value = customText !== undefined ? customText : defaultText;
 
 		return (
-			<View style={ styles[ 'block-library-more__container' ] }>
-				<View style={ styles[ 'block-library-more__sub-container' ] }>
-					<Text style={ styles[ 'block-library-more__left-marker' ] }>&lt;!--</Text>
-					<PlainText
-						style={ styles[ 'block-library-more__plain-text' ] }
-						value={ value }
-						multiline={ true }
-						underlineColorAndroid="transparent"
-						onChange={ this.onChangeInput }
-						placeholder={ defaultText }
-						isSelected={ this.props.isSelected }
-						onFocus={ onFocus }
-						onBlur={ onBlur }
-					/>
-					<Text style={ styles[ 'block-library-more__right-marker' ] }>--&gt;</Text>
-				</View>
+			<View key={ key } >
+				<PlainText
+					style={ styles[ 'block-library-more__text' ] }
+					value={ value }
+					multiline={ true }
+					underlineColorAndroid="transparent"
+					onChange={ this.onChangeInput }
+					placeholder={ defaultText }
+					isSelected={ this.props.isSelected }
+					onFocus={ onFocus }
+					onBlur={ onBlur }
+				/>
+			</View>
+		)
+	}
+
+	renderInner() {
+		return [
```

Comment:
Noticed that there are a few `key` params passed around. Is it perhaps because of the array defined here? If so, can we perhaps use https://reactjs.org/docs/fragments.html instead?

Reply:
on the second thought I decided we don't really need those `key` attributes so I removed them to save us from confusion. 
Because they are for optimization of re-rendering in the react subsystem, for example if you insert a new sibling at the beginning of the elements, react can decide NOT to re-render other siblings by comparing elements with the same key. But in our case we don't insert/remove elements dynamically so I think it is ok to get rid of `key`s.